src/mmw/js/src/draw/controllers.js

```diff
@@ -53,7 +53,7 @@ var DrawController = {
     },
 
     drawCleanUp: function() {
-        delete App.analyzeModel;
+        delete App.analyzeCollection;
```

Comment:
Done. Also moved the code from here, which gets executed every time `draw` is exited, to `ResetDrawView.resetDrawingState`. Now, the analysis results will be preserved if the user goes back to Draw but does not reset the drawing state, and will only be refreshed when the existing area of interest is cleared.